crypto/bip39.c

```diff
@@ -154,13 +154,32 @@ int mnemonic_to_entropy(const char *mnemonic, uint8_t *entropy) {
   if (bi != n * 11) {
     return 0;
   }
-  memcpy(entropy, bits, sizeof(bits));
+  memcpy(bits, result, sizeof(result));
```

Comment:
speaking of memzero, shouldn't even the original mnemonic_to_entropy() function perform a memzero on the "bits" array?

Reply:
ok, adding the call bellow as well